src/dms/core/EdFi.DataManagementService.Core/Middleware/ParsePathMiddleware.cs

```diff
@@ -98,35 +98,26 @@ public async Task Execute(PipelineContext context, Func<Task> next)
         }
 
         // Verify method allowed with/without documentUuid
-        switch (context.Method)
```

Comment:
The previous `case` statements were unusual having brackets, and EditorConfig and CSharpier had conflicting formatting thoughts. Changing to `if` statements was more compact anyway.